# New generation
int32 Generation

# New robots genes
int32[] RobotNumber
float32[] SensorActivation
float32[] LinearVelocity
float32[] AngularVelocity
float32[] RotationTime
float32[] SensorAngle
